package.json

```diff
@@ -18,17 +18,13 @@
   ],
   "license": "MIT",
   "main": "build/playcanvas.js",
-  "module": "build/playcanvas.mjs",
-  "types": "build/playcanvas.d.ts",
+  "module": "build/playcanvas/index.js",
```

Comment:
So I see the `types` property has been dropped from the `package..json`. How are types to be found by clients of the package now?

Reply:
So the unbundled types are now in the same directory as the unbundled so they are recognised when importing from there. If you were to use the UMD build the types are infered directly from playcanvas.d.ts since its name matches the package name. I might still include `types` just in case some issues arise using older versions of node or npm